assessments/projects/tikv/self-assessment.md

```diff
@@ -0,0 +1,335 @@
+# TiKV Security Self-Assessment
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Background](#background)
+  * [Architecture](#architecture)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+* [STRIDE Threat Model](#stride-threat-model)
+* [Questions about potential vulnerabilities](#questions-about-potential-vulnerabilities)
+
+## Metadata
+|   |  |
+| -- | -- |
+| Software | [TiKV](https://github.com/tikv/tikv/tree/master)  |
+| Security Provider | No  |
+| Languages | Rust, Java, Go, and C |
+| SBOM |Link to the dependencies file: TiKV does not currently generate an SBOM on release |
+| | |
+
+### Security links
+
+
+| Doc | url |
+| -- | -- |
+| Security file | https://github.com/tikv/tikv/blob/master/security/Security-Audit.pdf |
```

Comment:
Security file usually refers to security.md which consists of the governance process and procedures in finding and solving security issues. There is one provided by the team, please update https://github.com/tikv/tikv/blob/master/SECURITY.md for the security file and move the security audit report to another row in this table

Reply:
We have updated the table accordingly. Please refer to our new pull request #1207.